CMakeLists.txt

```
@@ -2,14 +2,27 @@ cmake_minimum_required (VERSION 3.1)
 project (IsoRender C)
 
 find_package(PNG REQUIRED)
```

with vcpkg this library is listed under a different name

```suggestion
find_package(libpng REQUIRED)
```